src/ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -210,6 +210,27 @@ public static class OAPHCreationHelperMixin
             return ret;
         }
 
+        static ObservableAsPropertyHelper<TRet> observableToProperty<TObj, TRet>(
```

Comment:
Formatting here seems quite a ways off when reviewing in GitHub - mixture of tabs and spaces?